### ISSUE TITLE
Consider changing tab order of relay icon in email input field

### ISSUE THREAD

**Issue by groovecoder** [mozilla/fx-private-relay#576]
From [AMO review](https://addons.mozilla.org/en-US/firefox/addon/private-relay/reviews/1575624/):

> once installed and that purple icon appears inside email login field its capturing the tab key so cannot simply type in emailid hit tab and enter password ... that purple widget should NOT capture the tab key when cursor is inside login(emailid) field ... Please fix

Consider changing the tab order of the add-on input button.

**Comment by tedschuh** [comment]
Should fix before GA.
